core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
 	gethCommon "github.com/ethereum/go-ethereum/common"
```

Comment:
One more dupe

Reply:
ugh...one moment, I will double check all of the files 🤦🏼